cpp/dolfinx/mesh/Geometry.h

```diff
@@ -64,7 +64,7 @@ class Geometry
   }
 
   /// Copy constructor
```

Comment:
Probably want to change `///` -> `//` so the deleted constructor doesn't get documented. Not sure what doxygen does with deleted constructors.